### PR TITLE
Document deprecating IS_CLIENT_ROLE searchable field

### PR DESCRIPTION
**Pull request by mhajas**

### CODE REVIEW

File: release_notes/topics/21_0_0.adoc [hunk 1]
```diff
@@ -37,4 +37,10 @@ See the link:{upgradingguide_link}[{upgradingguide_name}] for more details.
...
 |RSA1_5
-|===
\ No newline at end of file
+|===
+
+= New storage: IS_CLIENT_ROLE searchable field was deprecated
+
+The IS_CLIENT_ROLE searchable field from the RoleModel was deprecated. It
+should be replaced with the CLIENT_ID searchable field used with the operators
+EXISTS or NOT_EXISTS. See JavaDoc of Keycloak 21 for more details.
```
Comment (hmlnarik) [inline release_notes/topics/21_0_0.adoc]: Please use backticks for Java identifiers

File: release_notes/topics/21_0_0.adoc [hunk 2]
```diff
@@ -37,4 +37,10 @@ See the link:{upgradingguide_link}[{upgradingguide_name}] for more details.
 
 |https://www.w3.org/TR/2002/REC-xmlenc-core-20021210/Overview.html#rsa-1_5[rsa-1_5]
 |RSA1_5
-|===
\ No newline at end of file
+|===
+
+= New storage: IS_CLIENT_ROLE searchable field was deprecated
```
Comment (hmlnarik) [inline release_notes/topics/21_0_0.adoc]:
```suggestion
= New storage: `IS_CLIENT_ROLE` searchable field of `RoleModel` was deprecated
```

**Comment by mhajas** [comment]
@hmlnarik Thanks for the review, I updated the PR

**Review by hmlnarik — DISMISSED** [review]

**Review by hmlnarik — APPROVED** [review]

**Review by andymunro — APPROVED** [review]

**Comment by mhajas** [comment]
Thank you for the reviews @hmlnarik and @andymunro!

**Pull request merged**